app/build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
     testImplementation "androidx.test:core-ktx:1.4.0"
     testImplementation "org.robolectric:robolectric:4.8.1"
     testImplementation "com.google.dagger:hilt-android-testing:2.38.1"
-    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.5.2"
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.1"
```

Comment:
When you updated the version does the test continue working?

Reply:
Yes I checked and rechecked